inference_client/tasks/upscale.py

```diff
@@ -0,0 +1,223 @@
+                The path should end with the file extension of the output format, and could be either '.jpeg' or '.png'.
+                If both `output_path` and `output_format` are provided, the `output_format` will be ignored, and the
+                output image will be saved in the format of the file extension of `output_path`. Default: None.
+        :param kwargs: additional arguments to pass to the model.
+        """
+        ...
+
+    @overload
+    def upscale(
+        self,
+        *,
+        docs: Union[Iterable['Document'], 'DocumentArray'],
+        scale: Optional[str] = None,
+        **kwargs,
+    ):
+        """
+        Upscale image documents. The result will be stored in the `blob` attribute of the document in the format of
+        `output_format` specified in the `tags` attribute. If `output_path` is provided in the `tags` attribute, the
+        image will also be saved to the specified path.
+
+        :param docs: the image documents to upscale. The `output_format` and `output_path` should be provided in the
+                `tags` attribute of each document. The `output_format` specifies the format of the output image, could
+                be either `jpeg` or `png`. The `output_path` specifies the complete file path to save the output image
+                if provided in addition to returning it. The path should end with the file extension of the output
+                format, and could be either '.jpeg' or '.png'. If both `output_path` and `output_format` are provided,
+                the `output_format` will be ignored, and the output image will be saved in the format of the file
+                extension of `output_path`.
+        :param scale: the scale of the output image, if not provided, the image of the model output will be used. The
+                scale should be in the format of `width:height`, e.g. `100:200`. Both width and height should be
+                integers. If the width is 0, the input width is used for the output. If the height is 0, the input
+                height is used for the output. If one and only one of the values is -n with n >= 1, the scale filter
+                will use a value that maintains the aspect ratio of the input image, calculated from the other specified
+                dimension. After that it will, however, make sure that the calculated dimension is divisible by n and
+                adjust the value if necessary. If both values are -n with n >= 1, the behavior will be identical to both
+                values being set to 0 as previously detailed. Default: None.
+        :param kwargs: additional arguments to pass to the model
+        """
+        ...
+
+    @overload
+    def upscale(
+        self,
+        *,
+        docs: Optional[Union[Iterable['Document'], 'DocumentArray']] = None,
+        image: Optional[Union[str, bytes, 'ArrayType']] = None,
+        scale: Optional[str] = None,
+        output_format: str = 'jpeg',
```

Comment:
```suggestion
        image_format: str = 'jpeg',
```

as same as https://docarray.jina.ai/api/docarray.document.mixins.image/#docarray.document.mixins.image.ImageDataMixin.convert_image_tensor_to_blob